nano/node/network.cpp

```diff
@@ -4,13 +4,14 @@
 #include <numeric>
 #include <sstream>
 
-nano::network::network (nano::node & node_a, uint16_t port_a) :
-buffer_container (node_a.stats, nano::network::buffer_size, 4096), // 2Mb receive buffer
-resolver (node_a.io_ctx),
-node (node_a),
-udp_channels (node_a, port_a),
-tcp_channels (node_a),
-disconnect_observer ([]() {})
+nano::network::network (nano::node & node_a, uint16_t port_a)
+	: buffer_container (node_a.stats, nano::network::buffer_size, 4096)
+	, // 2Mb receive buffer
+	resolver (node_a.io_ctx)
```

Comment:
The placement of the comment looks a bit strange, perhaps just remove it (it seems inaccurate anyway?)

Reply:
Looks like it's accurate (512*4096) but seems prone to get out of sync, so can probably be removed.

Reply:
Updated